it/repositories/ApplicationRepositorySpec.scala

```diff
@@ -115,8 +116,11 @@ class ApplicationRepositorySpec extends MongoRepositorySpec {
         appStatus <- applicationRepo.findStatus(app.applicationId)
       } yield appStatus).futureValue
 
+      val hasCorrectStatusDate = Seconds.secondsBetween(DateTime.now(), applicationStatus.statusDate.get)
+        .isLessThan(Seconds.ONE)
```

Comment:
How reliable was this? Do you think a one second buffer is enough especially where mongo is networked etc (e.g. in build?)